fendermint/app/config/default.toml

```
@@ -50,3 +52,12 @@ host = "127.0.0.1"
 # The default port where the Ethereum API will listen to
 # JSON-RPC (POST) and WebSockets (GET) requests.
 port = 8545
+
+# IPC Parent finality related configuration
+[parent_finality]
+# The number of blocks behind chain head to propose parent finality.
+chain_head_delay = 10
+# The number of seconds to polling ipc agent for latest parent view
+polling_interval_secs = 10
+# The ipc agent url
+ipc_agent_url = "http://0.0.0.0:3030/json_rpc"
```

```suggestion
ipc_agent_url = "http://127.0.0.1:3030/json_rpc"
```

Although both work, I think this is more idiomatic; I only see 0.0.0.0 as a binding address for starting servers.